src/main/java/org/synyx/urlaubsverwaltung/application/web/ApplicationForLeaveFormViewController.java

```diff
@@ -140,7 +140,7 @@ public String newApplicationForm(
         }
 
         model.addAttribute("noHolidaysAccount", holidaysAccount.isEmpty());
-
+        model.addAttribute("applicationSettings", settingsService.getSettings().getApplicationSettings());
```

Comment:
Please define a specific attribute like `allowedHalfVacationDays` e.g. and add only this to the model. We do not want to give the persission to the view to have all settings, that are also not needed. 

You can add this like btw into the `prepareApplicationForLeaveForm` method. So you only have one place to add this.